initializr-service/src/main/resources/application.yml

```diff
@@ -1252,6 +1259,26 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+        - name: Codecentric Spring Boot Admin Server
+          bom: codecentric-spring-boot-admin
+          description: Provides the Spring Boot Admin Server API and UI
+          id: codecentric-spring-boot-admin-starter-server
+          artifactId: codecentric-spring-boot-admin-starter-server
+#          version: 1.5.7
+          groupId: de.codecentric
+          links:
+            - rel: reference
+              href: http://codecentric.github.io/spring-boot-admin/current/#getting-started
+        - name: Codecentric Spring Boot Admin Client
+          id: codecentric-spring-boot-admin-starter-client
+          bom: codecentric-spring-boot-admin
+          description: Registers applications with the Spring Boot Admin. Alternatively, Spring Boot Admin supports Spring Cloud DiscoveryClient implementations like Eureka or Consul.
+          artifactId: codecentric-spring-boot-admin-starter-client
+#          version: 1.5.7
+          groupId: de.codecentric
+          links:
+            - rel: reference
+              href: http://codecentric.github.io/spring-boot-admin/1.5.6/#getting-started
```

Comment:
should be http://codecentric.github.io/spring-boot-admin/current/#getting-started